src/api.js

```diff
@@ -82,7 +82,11 @@ export function ecrecover(hash, sign, v, malleabilityFlag) {
 }
 
 export function panic(msg) {
-    env.panic(msg)
+    if (msg !== undefined) {
```

Comment:
panicUtf8 / panicUtf16's interface is `panicUtf8(msg: bytes)`, and panic is `panic(msg?: any)`. So `panicUtf8/16` user must call with a msg in bytes, otherwise the behavior is undefined/not guaranteed.

For panic, user can pass any kind of object or nothing. If nothing is passed, quickjs will set msg as `undefined` (same behavior as nodejs), in this case we want to invoke host function `panic()`, instead of `panic(undefined)`